src/components/Modal/POIProposalModal/POIProposalModal.tsx

```diff
@@ -16,12 +15,14 @@ export function POIProposalModal({ ...props }) {
   const t = useFormatMessage()
 
   return (
-    <Modal {...props} size="tiny" className={TokenList.join(['ProposalModal', props.className])} closeIcon={<Close />}>
-      <Modal.Content className="ProposalModal__Title">
-        <Header>{t('category.poi_title')}</Header>
-        <Paragraph small>{t('modal.poi_proposal.description')}</Paragraph>
+    <Modal {...props} size="tiny" className="GovernanceContentModal ProposalModal" closeIcon={<Close />}>
+      <Modal.Content>
+        <div className="ProposalModal__Title">
+          <Header>{t('category.poi_title')}</Header>
+          <Paragraph small>{t('modal.poi_proposal.description')}</Paragraph>
+        </div>
       </Modal.Content>
-      <Modal.Content className="PoiProposalModal__Actions">
+      <div className="ProposalModel__Actions">
```

Comment:
I believe you meant `ProposalModal__Actions` instead of  `ProposalModel__Actions` (a x e)